src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,25 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+
+  @for $var from 0 through 5 {
+    & > :nth-child(#{$var + 1}) {
+      background-color: rgb(calc(255 - 51 * #{$var}), 0, 0);
+    }
```

Comment:
No need to overload specificity of selectors. You have class for `grid__cell` , use it.